src/Stratis.Bitcoin.Features.Api/NodeController.cs

```diff
@@ -71,6 +71,8 @@ public class NodeController : Controller
         /// <summary>An interface implementation used to retrieve unspent transactions.</summary>
         private readonly IGetUnspentTransaction getUnspentTransaction;
 
+        private readonly IInitialBlockDownloadState initialBlockDownloadState;
```

Comment:
This got added for the IBD check when calling node status from the masternode registration app.